CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Changed
+- tbd.
+
+## [1.0.0-beta0.1] - 2019-02-23
```

Comment:
Can you please add the link to compare releases to this release?
-> [1.0.0-beta0.1](https://github.com/superchargejs/framework/compare/v1.0.0-beta0...v1.0.0-beta0.1)